doc/user/bgp.rst

```diff
@@ -1587,6 +1588,8 @@ routing domain which is shared across all its sites. More complex routing
 topologies are possible through use of additional route-targets to augment the
 leaking of sets of routes in various ways.
 
+For direct VRF to VRF leaking the RD and RT are auto-derived.
```

Comment:
I propose alternate text here:

`When using the shortcut syntax for vrf-to-vrf leaking, the RD and RT are auto-derived.`

Reply:
done.